infra/modules/service/variables.tf

```diff
@@ -204,12 +204,30 @@ variable "enable_load_balancer" {
   default     = true
 }
 
-variable "enable_cdn" {
-  description = "Whether to enable a CDN for the service"
+variable "enable_alb_cdn" {
+  description = "Whether to enable an ALB origin CDN for the service. Cannot be enabled at the same time as the S3 CDN."
   type        = bool
   default     = false
 }
 
+variable "enable_s3_cdn" {
+  description = "Whether to enable a S3 origin CDN for the service. Cannot be enabled at the same time as the ALB CDN."
+  type        = bool
+  default     = false
+}
+
+variable "s3_cdn_bucket_name" {
+  description = "The name of the S3 bucket to use for the S3 CDN."
+  type        = string
+  default     = null
+}
+
+variable "s3_cdn_domain_name" {
+  description = "The domain name of the S3 bucket to use for the S3 CDN"
+  type        = string
+  default     = null
+}
+
```

Comment:
Doesn't need to be in this PR, but we'll want this value to be an env var the API gets.

The plan I had for this was just to swap out the s3 bucket for the CDN domain name on the fly in the API (eg. `s3://public-bucket-1234/path/to/file.txt` becomes `https://the-cdn-name/path/to/file.txt`)